### PR TITLE
Documentation for imputeByFD function

### PR DESCRIPTION
**Pull request by t201**
This is a documentation for imputeByFD function

[imputeByFD.txt](https://github.com/apache/systemml/files/4747335/imputeByFD.txt)

### CODE REVIEW

**Comment by kev-inn** [comment]
Are PRs #965 #964 just copies of this one? @t201 If so would it be possible to close them, maybe even with a note referencing this PR.

**Comment by t201** [comment]
Sure, will do that.

**Comment by j143** [comment]
Will review this tomorrow.

File: p-212 [hunk 1]
```diff
@@ -0,0 +1,30 @@
+## `imputeByFD` -Function
+Data imputation using functional dependencies is a probabilistic technique to fill the missing data inside a data frame.
+
+### Usage
+```r
+data_imputeFD = function(Data[String] x, Integer imputeData, Integer y, Double threshold)
+  return(Data[String] imputedData_X)
```
Comment (j143) [inline p-212:7]: Also, we need to make changes in .md file. See other PR for more info.

**Pull request closed without merging**